list/harvesterhci.io.management.cluster.vue

```diff
@@ -45,6 +41,17 @@ export default {
         },
       };
     },
+
+    rows() {
+      const inStore = this.$store.getters['currentProduct'].inStore;
+      const clusters = this.$store.getters[`${ inStore }/all`](HCI.CLUSTER);
+
+      return clusters.filter(c => c?.metadata?.labels?.[HCI_LABEL.HARVESTER_CLUSTER] === 'true' || c.provider === VIRTUAL_HARVESTER_PROVIDER);
```

Comment:
https://github.com/rancher/dashboard/issues/4273#issuecomment-926474073

@WuJun2016 What does `c.provider === VIRTUAL_HARVESTER_PROVIDER` mean? Is it a cluster provisioned by Harvester or it's a Harvester baremetal cluster?

Reply:
@yasker    it's a Harvester baremetal cluster.

Reply:
Hmm, then what does HCI_LABEL.HARVESTER_CLUSTER mean? Rancher UI team is checking if there is a way to know an imported cluster is Harvester or not, without adding a label on Virtualization Management from the Rancher side. cc @guangbochen

Reply:
From a Rancher perspective, they should use `provider=harvester` instead of any labels, this should only be used for the Harvester cluster only.